cep-XXXX.md

```diff
@@ -0,0 +1,265 @@
+- `version: str`. Normalized package version. It MUST comply with [CEP XX (Version strings)](https://github.com/conda/ceps/pull/132).
+- `build: str`. A string that helps disambiguate different variant builds of the same package version. It MUST comply with [CEP 26](./cep-0026.md). It SHOULD contain the `build_number` field at the end of the string, preceded by an underscore `_`. It MAY contain the hexadecimal string of the SHA1-hash of the key-sorted dictionary provided in `./info/hash_input.json`, preceded by `h`, usually trimmed to the first seven characters.
+- `build_number: int`. A non-negative integer representing the build number of the package.
+- `depends: list[str]`. Dependencies the package requires at runtime. Each string MUST be a valid `MatchSpec` positional string, as defined in [CEP XX (MatchSpec)](https://github.com/conda/ceps/pull/82).
+- `constrains: list[str]`. Dependencies the package is compatible with at runtime. These are not required, but if present, they must match the specifier. Each string MUST be a valid `MatchSpec` positional string, as defined in [CEP XX (MatchSpec)](https://github.com/conda/ceps/pull/82).
+- `subdir: str`. The target platform for this package, or `noarch` if platform-agnostic. It MUST comply with [CEP 26](./cep-0026.md).
+- `noarch: Literal['generic', 'noarch']`. Optional. When `subdir` is `noarch`, this field indicates the type of `noarch` package. It MUST be one of: `generic`, `python`.
+- `timestamp: int`. Starting time of the package build. It MUST be expressed as [Unix time](https://en.wikipedia.org/wiki/Unix_time) in milliseconds.
+- `track_features: str`. Space-separated string of unique identifiers.
+- `python_site_packages_path: str`. Site-packages path, as introduced in [CEP 20](./cep-0020.md).
+
+Additional keys MAY be present. The following ones are well-known but deprecated:
+
+- `arch: str`. The architecture the package is built for, as returned by Python's `platform.machine()`, or empty in the case of `subdir == 'noarch'`.
+- `platform: str`. The OS the package is built for. This is the first component of `subdir`, or empty in the case of `noarch`.
+- `features: str`. Space-separated string of identifiers of properties unique to this build.
+- `app: dict[str, Any]`. Metadata for Anaconda Navigator.
+- `preferred_env: str`.
+- `provides_features: dict[str, str]`.
+- `requires_features: dict[str, str]`.
+
+#### `./info/paths.json`
+
+Required.
+
+This file MUST list the contents of the package as a list of _path entries_, as described below. It MUST not list the contents of the `info/` folder.
+
+Each _path entry_ MUST be a dictionary that complies to the following schema:
+
+- `_path: str`. The path of the file, relative to the root of the package. It MUST use `/` as a path separator, even on Windows.
+- `path_type: Literal['hardlink', 'softlink', 'directory']`. Optional, defaults to `hardlink`. Type of path entry.
```

Comment:
Similar to above in `./info/index.json`, it would be good to standardize on a form to denote if a field is required or optional.